packages/kbn-securitysolution-utils/src/esql/get_index_list_from_esql_query.test.ts

```diff
@@ -14,7 +14,7 @@ jest.mock('@kbn/esql-utils', () => {
   };
 });
 
-const getIndexPatternFromESQLQueryMock = getIndexPatternFromESQLQuery as jest.Mock;
+const getIndexPatternFromESQLQueryMock = getIndexPatternFromESQLQueryDeprecated as jest.Mock;
```

Comment:
ℹ️ Here we need to use the async getIndexPatternFromESQLQuery instead but I prefer the code owners to handle it